docs/DevOps/Index.md

```diff
@@ -0,0 +1,15 @@
+# Introduction to DevOps
+
+This is the section dedicated to **DevOps**. 
+Within the following pages of this section, you will find more information about our DevOps practices, procedures and ways of working that facilitate cohesion and collaboration across the BHoM. 
+
+## What is DevOps?
+
+The word DevOps is a combination of the terms *development* and *operations*. The vision of DevOps is to have a smooth, streamlined and sophistical cycle for development, deployment and to provide improvements across the BHoM Community. 
```

Comment:
```suggestion
The word DevOps is a combination of the terms *development* and *operations*. The vision of DevOps is to have a smooth, streamlined and sophisticated cycle for development, deployment and improvements across the BHoM Community. 
```

Reply:
Definitely agree with changing to "sophisticated" but I think it would be better to add an oxford comma instead of removing "to provide".

Reply:
As discussed offline, that is fair and your point was well made so am happy to go with that suggestion 😄 👍